### PR TITLE
Fix several issues

### PR DESCRIPTION
**Pull request by tbruno**

### CODE REVIEW

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ After you provision the adapter, you must create an API to bind to the service.
 
 * <<create-api-AP, Create an API using Anypoint Platform>>
 +
-Create APIs using Anypoint Platform when your API specification files are larger than 550 KB. If you want to upload a REST API or an OAS asset type, use a `.json` or `.yaml` file. If you want to upload a SOAP API or WSDL asset type, use a `.wsdl` or `.xml` file.
+Create APIs using Anypoint Platform when your API specification files are larger than 550 KB. If you want to upload a REST API or an OAS asset type, use a `.json`, `raml`, `zip` or `.yaml` file. If you want to upload a SOAP API or WSDL asset type, use a `.wsdl` or `.xml` file.
```
Comment (dosten) [inline modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc]: Missing dots in raml and zip

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ After you provision the adapter, you must create an API to bind to the service.
 
 * <<create-api-AP, Create an API using Anypoint Platform>>
 +
-Create APIs using Anypoint Platform when your API specification files are larger than 550 KB. If you want to upload a REST API or an OAS asset type, use a `.json` or `.yaml` file. If you want to upload a SOAP API or WSDL asset type, use a `.wsdl` or `.xml` file.
+Create APIs using Anypoint Platform when your API specification files are larger than 550 KB. If you want to upload a REST API or an OAS asset type, use a `.json`, `raml`, `zip` or `.yaml` file. If you want to upload a SOAP API or WSDL asset type, use a `.wsdl` or `.xml` file.
```
Comment (dosten) [inline modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc]: Same here

**Review by dosten — APPROVED** [review]

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc]: Done! Thanks!

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 3]
```diff
@@ -25,13 +23,13 @@ nto-payments   customers   anypoint-service-mesh-adapter   1234    customers
 
 == Methods for Binding an API
 
-You can bind an API to a service using Anypoint Platform or Anypoint Service Mesh, based on your preference:
+You can bind an API to a service using API ID or Anypoint Service Mesh API name, based on your preference:
```
Comment (rddheath) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: "using API ID" should be "using an API ID".

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 4]
```diff
@@ -25,13 +23,13 @@ nto-payments   customers   anypoint-service-mesh-adapter   1234    customers
...
 
-* <<bind_the_service, Bind an API created using Anypoint Platform>>: If you created your API using Anypoint Platform, use this procedure.
-* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh>> : If you created your API using Anypoint Service Mesh, use this procedure.
+* <<bind_the_service, Bind an API referencing an API ID>>:  Use this procedure if you want to bind to the API using the API ID. This way is recommended if you created your API using Anypoint Platform.
```
Comment (rddheath) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Is "API ID" here the same as "the API's ID"? If so, the latter would be more clear.

Comment (rddheath) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: By "this way", do you mean "this procedure"?

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 5]
```diff
@@ -25,13 +23,13 @@ nto-payments   customers   anypoint-service-mesh-adapter   1234    customers
...
+* <<bind_using_ASM, Bind an API created using Anypoint Service Mesh API name>>: Use this procedure if you want to bind to the API using the Service Mesh API name. This way is recommended if you created your API using Anypoint Service Mesh.
```
Comment (rddheath) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Again, by "this way", do you mean "this procedure"?

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 6]
```diff
@@ -25,13 +23,13 @@ nto-payments   customers   anypoint-service-mesh-adapter   1234    customers
...
 
 [[bind_the_service]]
-== Bind an API Created Using Anypoint Platform
+== Bind an API referencing an API ID
```
Comment (rddheath) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Perhaps "Bind an API by using a reference to its ID"?

Comment (rddheath) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Perhaps "Bind an API that references an API ID"?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 7]
```diff
@@ -83,15 +81,18 @@ In your network, you might need to whitelist the hostnames and ports of external
...
+Anypoint Service Mesh supports Istio versions 1.6.0 through 1.6.8 and 1.7.0. You can install any of these Istio versions in your environment.
 
 === Prerequisites
 
 Before you begin, ensure that you download Istio using the https://istio.io/docs/setup/[Istio Documentation].
 
 === Install and Configure Istio
 
-To install and configure Istio for Anypoint Service Mesh, run the following command from your CLI:`$ istioctl install`.
+To install and configure Istio for Anypoint Service Mesh, run the following command from your terminal:
```
Comment (rddheath) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:92]: "Istio for Anypoint Service Mesh" sounds like the name of one product.
I think you mean this: "Istio for use with Anypoint Service Mesh".

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 8]
```diff
@@ -83,15 +81,18 @@ In your network, you might need to whitelist the hostnames and ports of external
 [[install_Istio]]
 == Download, Install, and Configure Istio
 
-You can install Istio 1.6.x (`1.6.0` to `1.6.8`) or 1.7.x in your environment to support Anypoint Service Mesh.
+Anypoint Service Mesh supports Istio versions 1.6.0 through 1.6.8 and 1.7.0. You can install any of these Istio versions in your environment.
```
Comment (rddheath) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: "Anypoint Service Mesh supports Istio versions 1.6.0 through 1.6.8 and 1.7.0."
I would change that to "Anypoint Service Mesh supports Istio versions 1.6.0 through 1.6.8 and version 1.7.0."

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 9]
```diff
@@ -59,9 +57,9 @@ For more information about which permissions to use, see xref:api-manager::org-c
 * Obtain the Cluster Administrator role to access the Kubernetes cluster (only required for installation process).
 * If you have configured an external identity provider for your organization, create a xref:obtain-connected-apps-credentials.adoc[Connected App] in Anypoint Platform with Exchange Contributors and Manage APIs Configuration roles. Anypoint Service Mesh does not support users residing on an external identity provider for tasks, such as creating and discovering APIs.
 
-== Ports, IPs, and Hostnames Whitelist Requirements
+== Ports, IPs, and Hostnames Allow list Requirements
 
-In your network, you might need to whitelist the hostnames and ports of external resources to allow Anypoint Service Mesh to communicate with the MuleSoft-managed online Anypoint Platform APIs and services.
+You might need to allow a list of hostnames and ports of external resources in your network. This helps you to enable Anypoint Service Mesh to communicate with the MuleSoft-managed online Anypoint Platform APIs and services:
```
Comment (rddheath) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: I might need to allow a list of hostnames and ports in my network?
Or do you mean that I might need to allow some servers to access various ports in my network, and that I can create a list of those servers and ports?
"This helps you to enable"? How about "Doing so enables"?

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc [hunk 10]
```diff
@@ -85,7 +83,7 @@ image::api-manager-status-for-service-mesh.png[85%, 85%]
 
 [[bind_using_ASM]]
 
-== Bind an API Created Using Anypoint Service Mesh
+== Bind an API created using Anypoint Service Mesh API name.
```
Comment (rddheath) [inline modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Should "API name" be at the end of this heading?
Why is there a period at the end?

**Review by rddheath — APPROVED** [review]

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:92]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

**Pull request merged**